snap/frontend/components/AccountListModalBox/AccountListModalBox.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import AccountListModalBox from '.';
+import testHelper from '../testHelper';
+import { screen } from '@testing-library/react';
+
+describe('components/AccountListModalBox', () => {
+	const generateAccountsState = numberOfAccounts => {
+		const accounts = {};
+
+		for (let index = 0; index < numberOfAccounts; index++) {
```

Comment:
Should this be a helper function to create X number of accounts?

Reply:
yeah, sound good to me.